docs/_data/tools.yaml

```
@@ -6425,7 +6425,7 @@
   demo: 'https://medium.com/%40unchase/how-to-generate-c-or-typescript-client-code-for-openapi-swagger-specification-d882d59e3b77'
   description: >-
     a Visual Studio 2017/2019 extension to generate C# (TypeScript) HttpClient 
-	(or C# Controllers) code for OpenAPI (formerly Swagger API) web service with NSwag.
+    (or C# Controllers) code for OpenAPI (formerly Swagger API) web service with NSwag.
```

```suggestion
    (or C# Controllers) code for OpenAPI web service with NSwag.
```

We can omit "formerly Swagger API" for our audience.